src/data/equiv/local_equiv.lean

```diff
@@ -200,6 +205,97 @@ protected def to_equiv : equiv (e.source) (e.target) :=
 
 lemma image_source_eq_target : e '' e.source = e.target := e.bij_on.image_eq
 
+/-- We say that `t : set β` is an image of `s : set α` under a local equivalence if
+any of the following equivalent conditions hold:
```

Comment:
```suggestion
any of the following equivalent conditions holds:
```

Reply:
@sgouezel I think it was correct before: "if any [...] hold" reads better to me.

Reply:
English is not my native language, so I have no (strong) opinion here.

Reply:
I maintain that the grammar here doesn't need to be corrected. I think the rule is that "conditions" is plural, so the form of "hold" must agree with that. For whatever it's worth, the number of google results for various phrases agrees with me (although less overwhelmingly than I thought): 
- ["if any of the following equivalent conditions holds" (2710 results)](https://www.google.com/search?q=%22if+any+of+the+following+equivalent+conditions+holds%22) vs ["if any of the following equivalent conditions hold" (4050 results)](https://www.google.com/search?q=%22if+any+of+the+following+equivalent+conditions+hold%22),  
- ["if any of these conditions holds" (19100 results)](https://www.google.com/search?q=%22if+any+of+these+conditions+holds%22) vs ["if any of these conditions hold" (45100 results)](https://www.google.com/search?q=%22if+any+of+these+conditions+hold%22) 
- ["if any of the following conditions holds" (34200 results)](https://www.google.com/search?q=%22if+any+of+the+following+conditions+holds%22) vs ["if any of the following conditions hold" (98500 results)](https://www.google.com/search?q=%22if+any+of+the+following+conditions+hold%22)

Reply:
English is not my native language either, so I'll leave it to you. Still, the question is not whether "conditions" is plural, but whether "any of the the conditions" is plural, and "any" is more singular in my mind (like, in "any of them"). A quick grammar googling shows that you can do whatever you want (English is a very flexible language :-):
```quote
"When none, neither, either and any are followed by "of + plural noun/pronoun," they are normally used with singular verbs in a formal style in British English. Plural verbs are common in informal British usage and generally in American English.
```
from https://english.stackexchange.com/questions/475028/is-any-of-them-used-with-a-singular-or-plural-verb

TLDR: don't merge my suggestion, it's fine as it is.